allocator.go

```diff
@@ -2,36 +2,37 @@ package allocator
 
 import (
 	"fmt"
-
 	"github.com/irfansharif/or-tools/cpsatsolver"
 )
 
+type NodeID int64
 type RangeID int64
+type Resource int
 
-type Range struct {
-	id   RangeID
-	rf   int
-	tags []string
+type ConstraintOption func(*Allocator)
```

Comment:
Rename this to simply "Option". The package name is `allocator`, so this would read as `allocator.Option`.

Reply:
Sounds good.